packages/runtime/src/plugins/useConfigInServerContext.ts

```diff
@@ -8,7 +9,7 @@ export interface ConfigInServerContextOptions {
 export function useConfigInServerContext({
   configContext,
 }: ConfigInServerContextOptions): GatewayPlugin {
-  return {
+  const configInServerContextPlugin: ServerAdapterPlugin = {
```

Comment:
Let's also deduplicate the logic here;
https://github.com/graphql-hive/gateway/blob/first-log/packages/runtime/src/createGatewayRuntime.ts#L1092
It causes to the (relatively expensive) object spread etc multiple times.

Reply:
Have you checked the commit I shared?